File: Kadai05b.java
//22D81011040E Miura Ryota 三浦僚太　2024年5月17日

public class Kadai05b{
    public static void main(String[] args){
        Dice2.showSum();

        Dice2 d1 = new Dice2();
        Dice2 d2 = new Dice2(52);
        Dice2 d3 = new Dice2("S");
        Dice2 d4 = new Dice2(3,"L",1000);

        Dice2.showSum();
    }    
}

File: kadai05b.cpp
/*kadai05b,5/15,22D8101040E,ryota miura*/
#include <iostream>
#include <vector>
#include <cstdlib>
int main(int argc, char* argv[]){
	if(argc != 1){
		std::string str = argv[1];
		int n = str.length();
	    int a = 0;
	    if(str[0] == '-' && str[1] == 'n' && n == 2){
		    a = 1;
	    }
	    for(int i = 1 + a; i < argc; i++){
		    if(i != 1 + a)std::cout << " " << argv[i];
		    if(i == 1 + a)std::cout << argv[i];
	    }
	    if(str[0] != '-' && str[1] != 'n' && n != 2){
		    std::cout << "\n";
	    }
	}
}
/*
g++ -o kadai05b kadai05b.cpp                                       
./kadai05b -n Clang GCC MSVC
Clang GCC MSVC*/
